SPEC.md

```diff
@@ -920,6 +924,9 @@ newline := See Table (All Newline White_Space)
 line-space := node-space | newline | single-line-comment
 // Whitespace within nodes, where newline-ish things must be esclined.
 node-space := ws* escline ws* | ws+
+
+// Version marker
+version := '/-' unicode-space* 'kdl-version' unicode-space* ('1' | '2')
```

Comment:
This should be followed by a line-space to make sure it’s always on its own line

Reply:
`line-space` doesn't require newlines. Perhaps it would be good to add `node-terminator` as in a regular node?

Reply:
`line-space` can be `node-space`, which can be `ws+`, which can be `unicode-space`, which doesn't mandate a newline before the node
So this should either be a `newline` or `unicode-space* newline`?

Reply:
Node terminator seems like a better idea!, though it allows `;`, so also doesn't mandate newlines?

Reply:
yeah, I think it's gonna need to be `unicode-space* newline.